onelogin_aws_cli/configuration.py

```diff
@@ -22,20 +22,35 @@ def __init__(self, config_file=None):
         if self.file is not None:
             self.load()
 
+    @property
+    def has_defaults(self) -> bool:
+        """True if the default section is non-empty"""
+        return len(self.items(self.default_section)) > 0
+
     @property
     def is_initialised(self) -> bool:
         """True if there is at least one section"""
-        return (len(self.sections()) > 0)
+        return (len(self.sections()) > 0) or self.has_defaults
 
     def load(self):
         self.read_file(self.file)
+        # For backwards compatibility, we check if they have a default
+        # section instead of a defaults section and do a little switcheroo as
+        # needed.
+        if self.has_section('default') and not self.has_defaults:
+            print("It looks like you're using a the deprecated 'default' "
+                  "section.\nConsider renaming the section to 'defaults'.")
+            self.default_section = 'default'
```

Comment:
In regards to whether this is still relevant, it depends on how we want to use `default`. I will write some bits in the config section of the README and we can regroup and see if it is needed. Sound ok?

Reply:
Sure this was just in response to the reported issue; upon investigating I realized that until recently initialization was writing to a profile named 'default' so it seems like we should support it (or do a major version bump).

Reply:
Ah right. Your call. At the very least, I'll write details for the configuration behaviour. If you do close it, we can always come back to this branch.

Reply:
Sounds good, thanks. I'll pull the default swizzling into a separate PR and we can discuss there.